### PR TITLE
Enable session

### PR DESCRIPTION
**Pull request by beeguy74**
draft for #7 

* [`src/models.cairo`](diffhunk://#diff-a206f0ceecae52777c10750f7655aefdb31f5830f57a3ce10e389379d731cb69L44-R62): Modified the `Counter` struct
* [`src/systems/actions.cairo`](diffhunk://#diff-e2b52ebfd6509d27acc97b614e28ef7802a4b3752cf850eee27e4aebb1c681bfL68-R90): Updated the `create_lobby` function to properly read and write the `Counter` model, and adjusted the session ID handling.
* [`src/systems/actions.cairo`](diffhunk://#diff-e2b52ebfd6509d27acc97b614e28ef7802a4b3752cf850eee27e4aebb1c681bfL174-R179): Refined the `get_session_id` function to use the updated `Counter` model.
* [`src/tests/test_world.cairo`](diffhunk://#diff-0b36b09bd228c357b5016f0ba4534cd0b81823d28c6577cbbee14ce00f980307L9-R9): Tested all methods of `Counter` model, `session_id` for multiple sessions, and `get_session_id` action 

@gianalarcon

### CODE REVIEW

**Comment by beeguy74** [comment]
all test are  :ok_hand:
image

File: dojo-starter/src/models.cairo [hunk 1]
```diff
@@ -41,18 +41,25 @@ pub struct Player {
...
+    fn get_value(self: Counter) -> u64 {
+        self.value
+    }
 
-        let id = self.nonce;
+    fn increment(ref self: Counter) -> () {
+        self.value += 1;
+    }
+
+    fn decrement(ref self: Counter) -> () {
```
Comment (bitfalt) [inline dojo-starter/src/models.cairo:57]: In which cases would we use the decrement function?

**Review by bitfalt — CHANGES_REQUESTED** [review]
lgtm! Just answer the question and depending on the answer we'll merge it!

File: dojo-starter/src/models.cairo (hunk 1 above)
Comment (beeguy74) [reply dojo-starter/src/models.cairo:57]: The `Сounter` model can be used for other mechanics in the game if you set a different `global_key`, I thought that would be useful.

Comment (beeguy74) [reply dojo-starter/src/models.cairo:57]: @bitfalt  Hi, Daniel, can you merge PR, if you're ok with my answer?

Comment (bitfalt) [reply dojo-starter/src/models.cairo:57]: Perfect, I'll merge your PR. Thanks for thinking towards the future.

**Review by bitfalt — APPROVED** [review]

**Pull request merged**